packages/styles/src/functions/_tokens.scss

```diff
@@ -0,0 +1,41 @@
+@use 'sass:map';
+@use 'sass:string';
+
+/**
+ * @function get($keyName, $m: $map)
+ * Gets token-key, normalizes and resolved it with the $map or de one specified in the second parameter.
+ *
+ * @param {string} $keyName - The token key to get (with or without the "post" prefix).
+ * @param {map} $m - The map to get the token key from.
+ *                   The $map variable can be specified through the "with" keyword in the import statement:
+ *                   @use '../functions/tokens' as tokens with ($map: namespace.$my-default-map);
```

Comment:
```suggestion
 *                   @use '../functions/tokens' with ($map: namespace.$my-default-map);
```

Reply:
Nope. This must stay, as we want to call `tokens.get()`

Reply:
Yes, but the file is already called "tokens"